src/tlo/methods/rti.py

```diff
@@ -4017,12 +4131,13 @@ def apply(self, person_id, squeeze_factor):
             df.loc[person_id, 'rt_injuries_to_cast'].clear()
             df.loc[person_id, 'rt_date_death_no_med'] = pd.NaT
         else:
-            self.sim.modules['RTI'].schedule_hsi_event_for_tomorrow(self)
-            if pd.isnull(df.loc[person_id, 'rt_date_death_no_med']):
-                df.loc[person_id, 'rt_date_death_no_med'] = self.sim.date + DateOffset(days=7)
-            logger.debug(key='rti_general_message',
-                         data=f"Person {person_id} has {fracturecastcounts + slingcounts} fractures without treatment"
-                         )
+            if self._number_of_times_this_event_has_run < self._maximum_number_times_event_should_run:
+                self.sim.modules['RTI'].schedule_hsi_event_for_tomorrow(self)
+                if pd.isnull(df.loc[person_id, 'rt_date_death_no_med']):
+                    df.loc[person_id, 'rt_date_death_no_med'] = self.sim.date + DateOffset(days=7)
```

Comment:
I think this needs to be un-indented, so that it's not inside the clause to do with the number of times the HSI has run. i.e. this need to happen, irrespective of whether or not we're rescheduling the HSI